### PR TITLE
Add examples to Ping explore and Glean ping explore (fixes #107)

### LINKED ISSUE #107: Add example queries to Ping and Glean Ping explores
Looker lets you have [quick start queries](https://docs.looker.com/reference/explore-params/query) for explores, that automatically select some fields for an easy query for someone to build off of.

We should enable a few of these for Ping and Glean Ping explores. Some options could be:
- Ping count over the past 28 days
- Client Counter per top-N countries over the past 28 days (might be a little tough to determine top-N, but could filter on e.g. tier-1)
- Ping count per-normalized app version, past 6 months

### PR DESCRIPTION
**Pull request by scholtzan**
Adds a few example queries to ping explores and Glean ping explores, based on the suggestions in https://github.com/mozilla/lookml-generator/issues/107

fixes https://github.com/mozilla/lookml-generator/issues/107

### CODE REVIEW

File: generator/explores/ping_explore.py [hunk 1]
```diff
@@ -22,6 +41,7 @@ def _to_lookml(self, v1_name: Optional[str]) -> List[Dict[str, Any]]:
                 "always_filter": {
                     "filters": self.get_required_filters("base_view"),
                 },
+                "queries": deepcopy(PingExplore.queries),
```
Comment (acmiyaguchi) [inline generator/explores/ping_explore.py:44]: Why the deep copy?

**Review by acmiyaguchi — APPROVED** [review]
LGTM for starter queries.

File: generator/explores/ping_explore.py (hunk 1 above)
Comment (scholtzan) [reply generator/explores/ping_explore.py:44]: Good question, maybe @fbertsch can answer?
I've simply copied it from other existing example queries: https://github.com/mozilla/lookml-generator/blob/8db3ebc21e4daaeb861f75e943cc98e6c8d3ce16/generator/explores/events_explore.py#L52

Comment (scholtzan) [reply generator/explores/ping_explore.py:44]: I talked to Frank about this. Apparently, the `lkml` library that we are using mutates the data.

**Pull request merged**